pyaedt/application/Design.py

```diff
@@ -516,6 +516,11 @@ def __init__(self, design_type, project_name=None, design_name=None, solution_ty
         self.project_datasets = self._get_project_datasets()
         self.design_datasets = self._get_design_datasets()
 
+        def __del__(self, key):
+            """Implement destructor with array name or index."""
+            object_to_delete = self._variable_manager(key)
+            del object_to_delete
```

Comment:
This should be simplified with
```suggestion
            del self._variable_manager[key]
```

Reply:
I agree. we should follow @akaszynski suggestion